### PR TITLE
Issues #78

### LINKED ISSUE #78: System.FormatException when using HubSpotContactClient.ListAsync for models with a DateTime
When running the following code:
```
var customers = await
                        _hubSpotApi.ListAsync<ContactListHubSpotEntity<HubspotCustomer>>(new ContactListRequestOptions
                        {
                            ContactOffset = offset,
                            NumberOfContactsToReturn = _hubspotConfig.BatchSize,
                            PropertiesToInclude = _propertiesToInclude
                        });
```

I get the following exception:

```
System.FormatException
String '1594252800000' was not recognized as a valid DateTime  

  at System.DateTimeParse.Parse(ReadOnlySpan`1 s, DateTimeFormatInfo dtfi, DateTimeStyles styles)
   at System.Convert.ToDateTime(String value, IFormatProvider provider)
   at System.String.System.IConvertible.ToDateTime(IFormatProvider provider)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at Skarp.HubSpotClient.Core.Requests.RequestDataConverter.ConvertSingleEntity(ExpandoObject dynamicObject, Object dto)
   at Skarp.HubSpotClient.Core.Requests.RequestDataConverter.FromHubSpotListResponse[T](ExpandoObject dynamicObject)
   at Skarp.HubSpotClient.Core.Requests.RequestSerializer.DeserializeListEntity[T](String json)
   at Skarp.HubSpotClient.Core.HubSpotBaseClient.<ListAsync>b__11_0[T](String responseData)
   at Skarp.HubSpotClient.Core.HubSpotBaseClient.<SendRequestAsync>d__17`1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Skarp.HubSpotClient.Core.HubSpotBaseClient.<ListAsync>d__11`1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Skarp.HubSpotClient.Contact.HubSpotContactClient.<ListAsync>d__7`1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at MyProject.HubSpotService.<DownloadCustomers>d__9.MoveNext()
```

My HubspotCustomer class has a DateTime property.
The issue seems to be that the hubspot client is not parsing the returned date correctly.

### PR DESCRIPTION
**Pull request by LeeStevens318**
Provide fix for DateTime and DateTimeOffset (and nullables) 

Closes #78

### CODE REVIEW

File: src/Core/Requests/RequestDataConverter.cs [hunk 1]
```diff
@@ -289,9 +290,24 @@ internal object ConvertSingleEntity(ExpandoObject dynamicObject, object dto)
...
+                    // property type
+                    var pt = targetProp.PropertyType;
+                    var val = dynamicValue.ToString();
+
+                    // skip any nullable properties with a empty value
+                    if (Nullable.GetUnderlyingType(pt) != null && string.IsNullOrEmpty(val))
+                        continue;
+
+                    // convert to DateTime/DateTime?
+                    if ((pt == typeof(DateTime) || pt == typeof(DateTime?)) && long.TryParse(val, out var milliseconds))
```
Comment (nover) [inline src/Core/Requests/RequestDataConverter.cs:304]: **question** are we sure that the timestamp coming back from HubSpot is in `ms` and not in `s` ?

File: src/hubspot-client.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
-    <Version>0.19.2</Version>
+    <Version>0.19.3</Version>
```
Comment (nover) [inline src/hubspot-client.csproj:4]: **issue** we should keep version bumps out of MRs.

We handle that on master when releasing :)

**Review by nover — CHANGES_REQUESTED** [review]
Overall good, a few nitpicks

File: src/Core/Requests/RequestDataConverter.cs (hunk 1 above)
Comment (LeeStevens318) [reply src/Core/Requests/RequestDataConverter.cs:304]: Yes, i checked doc: [HubSpot API endpoints accept UNIX formatted timestamps in milliseconds.](https://legacydocs.hubspot.com/docs/faq/how-should-timestamps-be-formatted-for-hubspots-apis)

File: src/hubspot-client.csproj (hunk 2 above)
Comment (LeeStevens318) [reply src/hubspot-client.csproj]: reverted

**Review by LeeStevens318 — COMMENTED** [review]
Reverted version and added comment around ms vs s

**Review by nover — APPROVED** [review]

**Comment by nover** [comment]
@LeeStevens318 I'm thinking a minor release given it's a new feature

**Pull request merged**